apex/contrib/optimizers/distributed_fused_adam.py

```diff
@@ -2178,31 +2178,46 @@ def _local_step_with_param_remainders(
     def state_dict(
         self,
         *,
-        v1_format: bool = False,
+        state_dict_format: Optional[Any] = None,
```

Comment:
I'm not quite sure but shouldn't the annotation be

```suggestion
        state_dict_format: Optional[int] = None,
```
?

Reply:
My thought is that the format could also be a `str`, but I may be overthinking it.